### ISSUE TITLE
Tailwind 1.7.2 does not work with Hugo

### ISSUE THREAD

**Issue by bep** [gohugoio/hugo#7581]
I claim this is a bug upstream, see https://github.com/tailwindlabs/tailwindcss/issues/2203

But keep a tracking issue here for people that stumble upon the same as me.

A uglish workaround is to do something like this:

```html
{{ $styles := resources.Get "css/styles.css" }}
{{ $styles = $styles | resources.PostCSS $options }}
{{/* workaround for https://github.com/tailwindlabs/tailwindcss/issues/2203 */}}
{{ $stylesContent := $styles.Content }}
{{ $stylesContent = $stylesContent | replaceRE "(?m)^risk -.*$" "" }}
{{ $stylesContent = $stylesContent | replaceRE "(?m)^warn -.*$" "" }}
{{ $styles = $stylesContent | resources.FromString "css/styles.css" }}
```

**Comment by jmooring** [comment]
Resolved with Tailwind 1.7.3.

**Comment by bep** [comment]
Confirmed, thanks.
